easyfix-messages/src/fields/basic_types.rs

```diff
@@ -852,12 +852,16 @@ impl UtcTimestamp {
         }
     }
 
+    fn timestamp_from_secs_and_nsecs(secs: i64, nsecs: u32) -> DateTime<Utc> {
+        Utc.from_utc_datetime(&DateTime::from_timestamp(secs, nsecs).unwrap().naive_utc())
```

Comment:
I'm not sure but isn't just `DateTime::from_timestamp` enough?

Reply:
You are right. Sorry about that: I am new to Rust :(